spec/features/create_etd_spec.rb

```diff
@@ -0,0 +1,235 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Create a new ETD', type: :feature do
+  let(:etd_base_url) { 'etd-stage.stanford.edu' }
+  # dissertation id must be unique; D followed by 9 digits, e.g. D123456789
+  let(:dissertation_id) { format('D%09d', Kernel.rand(1..999_999_999)) }
+  let(:dissertation_title) { 'Integration Testing of ETD Processing' }
+  let(:xml_from_registrar) do
+    # see https://github.com/sul-dlss/hydra_etd/wiki/Data-Creation-and-Interaction#creating-new-etd-records
+    <<-XML
+    <DISSERTATION>
+      <dissertationid>#{dissertation_id}</dissertationid>
+      <title>#{dissertation_title}</title>
+      <type>Dissertation</type>
+      <vpname>Patricia J. Gumport</vpname>
+      <readerapproval>Not Submitted</readerapproval>
+      <readercomment> </readercomment>
+      <readeractiondttm> </readeractiondttm>
+      <regapproval>Not Submitted</regapproval>
+      <regactiondttm> </regactiondttm>
+      <regcomment> </regcomment>
+      <documentaccess>Yes</documentaccess>
+      <schoolname>School of Medicine</schoolname>
+      <degreeconfyr>2020</degreeconfyr>
+      <reader>
+        <sunetid>dedwards</sunetid>
+        <name>Edwards, Doris</name>
+        <type>int</type>
+        <univid>05358772</univid>
+        <readerrole>Doct Dissert Advisor (AC)</readerrole>
+        <finalreader>Yes</finalreader>
+      </reader>
+      <reader type="int">
+        <univid>05221995</univid>
+        <sunetid>rkatila</sunetid>
+        <name>Katila, Riitta</name>
+        <readerrole>Doct Dissert Reader (AC)</readerrole>
+        <finalreader>No</finalreader>
+      </reader>
+      <univid>05543256</univid>
+      <sunetid>dkelley</sunetid>
+      <name>Kelley, DeForest</name>
+      <career code="MED">Medicine</career>
+      <program code="MED">Medical</program>
+      <plan code="ANT">Neurology</plan>
+      <degree>PHD</degree>
+    </DISSERTATION>
+    XML
+  end
+
+  # may need to increase this: file uploads and submission to registrar
+  # Capybara.default_max_wait_time = 5
+
+  # See https://github.com/sul-dlss/hydra_etd/wiki/End-to-End-Testing-Procedure
+  it do
+    # registrar creates ETD in hydra_etd application by posting xml
+    resp_body = simulate_registrar_post(xml_from_registrar)
+    prefixed_druid = resp_body.split.first
+    expect(prefixed_druid).to start_with('druid:')
+    puts "dissertation id is #{dissertation_id}"
```

Comment:
Remove these puts please

Reply:
I find these 3 puts statements super helpful, especially since we can't really track it all the way through ETD submission.   What's the harm?

I can certainly comment them out - can you explain your objection?

Reply:
(it's also handy if you need to find an ETD on stage)

Reply:
Not a big deal, I thought these were leftover from developing the script. Just the very minor concern that this will clutter the rspec output.  Should we have a log file instead?

Reply:
I'll leave them in but comment them out.